chart/pdb-values.yaml

```diff
@@ -1,2 +0,0 @@
-# Will trigger creation of pdb
-replicaCount: 2
```

Comment:
Should we keep those in the ci directory, along with a blank default-values.yaml?